merchant-app/app/src/main/java/com/googleinterns/smb/model/Merchant.java

```diff
@@ -93,6 +95,7 @@ private Merchant() {
         email = user.getEmail();
         photoUri = user.getPhotoUrl();
         numProducts = getStoredNumProducts();
+        latLng = new LatLng(23.012265, 72.587970);
```

Comment:
Remove this hardcoding

Reply:
will add setting page functionality for getting merchant location later on, adding as issue for now